src/geo/projection/globe_util.js

```diff
@@ -337,17 +326,37 @@ export function aabbForTileOnGlobe(tr: Transform, numTiles: number, tileId: Cano
     return new Aabb(cornerMin, cornerMax);
 }
 
-export function globeTileLatLngCorners(id: CanonicalTileID): [[number, number], [number, number]] {
-    const tileScale = 1 << id.z;
-    const left = id.x / tileScale;
-    const right = (id.x + 1) / tileScale;
-    const top = id.y / tileScale;
-    const bottom = (id.y + 1) / tileScale;
+function tileCornersInMercator(tileId: CanonicalTileID, numTiles: number = 1): [number, number, number, number] {
+    const tileScale = numTiles / (1 << tileId.z);
```

Comment:
I haven't noticed a code path where do we use `tileCornersInMercator` with the second parameter. It might be best to inline:
```suggestion
function tileCornersInMercator(tileId: CanonicalTileID): [number, number, number, number] {
    const tileScale = 1.0 / (1 << tileId.z);
```

Reply:
Agreed! I was previously using that in https://github.com/mapbox/mapbox-gl-js/pull/11988 but I think the function makes much more sense without it.